src/main/java/org/kuali/kfs/kim/impl/role/RoleServiceImpl.java

```diff
@@ -1769,6 +1770,9 @@ public RoleMember assignPrincipalToRole(
         newRoleMember.setMemberId(principalId);
         newRoleMember.setType(MemberType.PRINCIPAL);
 
+        DateTimeService dateTimeService = SpringContext.getBean(DateTimeService.class);
```

Comment:
`RoleServiceImpl` already has a `dateTimeService` field, so you don't need to manually retrieve the service.